ansible/roles/operator-pipeline/templates/openshift/tasks/merge-registry-credentials.yml

```diff
@@ -0,0 +1,56 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: merge-registry-credentials
+spec:
+  params:
+    - name: cert_project_docker_config_json
+      description: Encrypted project's docker_config_json that contains private
+        registry tokens
+  workspaces:
+    - name: registry-credentials
+      description: Registry tokens provided as an input of a pipeline
+    - name: registry-credentials-all
+      description: Global workspace for storing all registry access tokens
+    - name: gpg-key
+      description: GPG private key that is used for decryption of user's tokens
+  steps:
+    - name: get-and-decrypt-project-tokens
+      image: quay.io/redhat-isv/operator-pipelines-images:latest
+      workingDir: $(workspaces.registry-credentials-all.path)
+      script: |
+        #! /usr/bin/env bash
+        set -e
```

Comment:
Please add a big warning comment about not setting `-x`.

Reply:
Fixed